app/src/main/java/com/internship/firstweekapp/ui/game/GameFragmentViewModel.kt

```diff
@@ -0,0 +1,8 @@
+package com.internship.firstweekapp.ui.game
+
+import com.internship.firstweekapp.arch.BaseViewModel
+
+class GameFragmentViewModel : BaseViewModel() {
+    var isUserFirst = true
```

Comment:
thinking it in another way
not userfirst or user cressed
create base class player
there is two derived class - Human and Computer
this class should store cross or square enum